src/main/java/teammates/common/util/NationalityHelper.java

```diff
@@ -0,0 +1,284 @@
+        addToNationalitiesList("Omani");
+        addToNationalitiesList("Pakistani");
+        addToNationalitiesList("Palauan");
+        addToNationalitiesList("Panamanian");
+        addToNationalitiesList("Papua New Guinean");
+        addToNationalitiesList("Paraguayan");
+        addToNationalitiesList("Peruvian");
+        addToNationalitiesList("Polish");
+        addToNationalitiesList("Portuguese");
+        addToNationalitiesList("Qatari");
+        addToNationalitiesList("Romanian");
+        addToNationalitiesList("Russian");
+        addToNationalitiesList("Rwandan");
+        addToNationalitiesList("Saint Lucian");
+        addToNationalitiesList("Salvadoran");
+        addToNationalitiesList("Samoan");
+        addToNationalitiesList("San Marinese");
+        addToNationalitiesList("Sao Tomean");
+        addToNationalitiesList("Saudi");
+        addToNationalitiesList("Scottish");
+        addToNationalitiesList("Senegalese");
+        addToNationalitiesList("Serbian");
+        addToNationalitiesList("Seychellois");
+        addToNationalitiesList("Sierra Leonean");
+        addToNationalitiesList("Singaporean");
+        addToNationalitiesList("Slovakian");
+        addToNationalitiesList("Solomon Islander");
+        addToNationalitiesList("Somali");
+        addToNationalitiesList("South African");
+        addToNationalitiesList("South Korean");
+        addToNationalitiesList("Spanish");
+        addToNationalitiesList("Sri Lankan");
+        addToNationalitiesList("Sudanese");
+        addToNationalitiesList("Surinamer");
+        addToNationalitiesList("Swazi");
+        addToNationalitiesList("Swedish");
+        addToNationalitiesList("Swiss");
+        addToNationalitiesList("Syrian");
+        addToNationalitiesList("Taiwanese");
+        addToNationalitiesList("Tajik");
+        addToNationalitiesList("Tanzanian");
+        addToNationalitiesList("Thai");
+        addToNationalitiesList("Togolese");
+        addToNationalitiesList("Tongan");
+        addToNationalitiesList("Tobagonian");
+        addToNationalitiesList("Tunisian");
+        addToNationalitiesList("Turkish");
+        addToNationalitiesList("Tuvaluan");
+        addToNationalitiesList("Ugandan");
+        addToNationalitiesList("Ukrainian");
+        addToNationalitiesList("Uruguayan");
+        addToNationalitiesList("Uzbekistani");
+        addToNationalitiesList("Venezuelan");
+        addToNationalitiesList("Vietnamese");
+        addToNationalitiesList("Welsh");
+        addToNationalitiesList("Yemenite");
+        addToNationalitiesList("Zambian");
+        addToNationalitiesList("Zimbabwean");
+        addToNationalitiesList("Other");
+    }
+    
+    /**
+     * This prevents instantiation. This is necessary because this is a utility class.
+     */
+    private NationalityHelper() {
+    }
+
+    /**
+     * Adds the nationalities to the NATIONALITIES list.
+     */
+    public static void addToNationalitiesList(String...nationalities) {
```

Comment:
Three changes possible here:
1. The method takes in varargs, but the usages are all with single String. Either make it accept just a single String, or keep it varargs but just invoke this method once with all the possible nationalities as the argument.
2. If you decide for the latter in (1), there should be a space after the three dots, i.e `String... nationalities`. I'm surprised that CheckStyle did not catch this.
3. Make it `private`; it does not seem to be used anywhere else.